autoload/lsc/complete.vim

```diff
@@ -125,8 +125,9 @@ function! s:SuggestCompletions(completion) abort
     let base = getline('.')[start - 1:col('.') - 2]
     let suggestions = s:FindSuggestions(base, a:completion)
   endif
-  setl completeopt-=longest
-  setl completeopt+=menu,menuone,noinsert,noselect
+  if (g:lsc_auto_completeopt == 1)
+    setl completeopt=menuone,noinsert,noselect
```

Comment:
The `+=` was intentional so that the `preview` option could remain untouched.

Reply:
It seems the completeopt keep growing under some condition. For example if I use vim-lsc and asyncomplete.vim (It's also modify the completeopt, https://github.com/prabirshrestha/asyncomplete.vim ) together.